Cable/CxxTypes/cxxUtils.h

```diff
@@ -29,7 +29,7 @@
 #endif
 
 // Visual C++ for-loop scoping hack.
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && _MSC_VER < 1700
 #ifndef for
 #define for if(false) {} else for
 #endif
```

Comment:
This was back from the VS 6 days (`_MSC_VER < 1300`).  We can drop the whole block.